Discord/src/bot/DraftBotDiscordMetrics.ts

```diff
@@ -0,0 +1,67 @@
+const client = require("prom-client");
+
+export const draftBotMetricsRegistry = new client.Registry();
+
+export abstract class DraftBotDiscordMetrics {
+	private static packetsTimeHistogram = new client.Histogram({
+		name: "discord_packets_time",
+		help: "Histogram of packets times",
+		labelNames: ["packet"],
+		registers: [draftBotMetricsRegistry],
+		buckets: [
+			0.1,
+			0.2,
+			0.3,
+			0.4,
+			0.5,
+			0.6,
+			0.7,
+			0.8,
+			0.9,
+			1,
+			1.5,
+			2,
+			2.5,
+			3,
+			3.5,
+			4,
+			4.5,
+			5,
+			6,
+			7,
+			8,
+			9,
+			10
```

Comment:
WOW, y'a pas moyen de le faire plus proprement ?

Reply:
ça ne me choque pas, ce ne sera pas forcément plus lisible avec des for qui s'enchaînent

Reply:
Le problème c'est la règle eslint qui veut 3 éléments par ligne max

Reply:
ce serait déjà mieux qu'actuellement

Reply:
Avec des for ça donne ça
```ts
/*
 * The time buckets for the histogram are defined here
 * [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1, 1.5, 2, 2.5, 3, 3.5, 4, 4.5, 5, 6, 7, 8, 9, 10]
 */
const packetsTimeBuckets: number[] = [];
for (let i = 0.1; i < 1; i += 0.1) {
	packetsTimeBuckets.push(i);
}
for (let i = 1; i < 5; i += 0.5) {
	packetsTimeBuckets.push(i);
}
for (let i = 5; i <= 10; i += 1) {
	packetsTimeBuckets.push(i);
}
```